p2p/libp2p/netMessenger.go

```diff
@@ -343,6 +375,18 @@ func (netMes *networkMessenger) printLogsStats() {
 	}
 }
 
+func (netMes *networkMessenger) checkExternalLoggers() {
+	for {
+		select {
+		case <-netMes.ctx.Done():
+			return
+		case <-time.After(secondsBetweenExternalLoggersCheck * time.Second):
```

Comment:
make the const directly equalt with 20 * time.Second - and the computation is not needed here.

Reply:
done